server/models/migrations/next/migrate.sql

```diff
@@ -5,3 +5,38 @@ ALTER TABLE `stock_consumption` MODIFY `period_id` MEDIUMINT(8) UNSIGNED NOT NUL
 ALTER TABLE `stock_consumption` ADD CONSTRAINT `fk_inventory_uuid_sc` FOREIGN KEY (`inventory_uuid`) REFERENCES `inventory` (`uuid`);
 ALTER TABLE `stock_consumption` ADD CONSTRAINT `fk_depot_uuid_sc` FOREIGN KEY (`depot_uuid`) REFERENCES `depot` (`uuid`);
 ALTER TABLE `stock_consumption` ADD CONSTRAINT `fk_period_id_sc` FOREIGN KEY (`period_id`) REFERENCES `period` (`id`);
+/*
+ * DATABASE CHANGES FOR VERSION 1.6.0 TO 1.7.0 
+ */
+
+/*
+* Pricelist importation
+  by Jeremielodi
+  2019-10-16
+*/
+
+DROP PROCEDURE IF EXISTS importPriceListItem;
```

Comment:
No need to add procedures to the migration script.  We rebuild all procedures when we do an upgrade anyway.